### ISSUE TITLE
Create plugins folder if does not exist

### ISSUE THREAD

**Issue by matveevandrey** [SpookOz/zabbix-wininventory#3]
I'd suggest to modify poweshell download command as following:

` powershell.exe -NoProfile -ExecutionPolicy Bypass -command Invoke-WebRequest -Uri https://git.io/JUTv4 -OutFile ( New-Item -Path "$Env:Programfiles\'Zabbix Agent\plugins\get-inventory-v5.ps1'" -itemtype file -Force )`

P.S. Does not work on Windows 2012R2/Windows2016 (plugins folder is successfully creating but ps1 script downloading fails)

**Comment by SpookOz** [comment]
Thanks. Maybe that's because Powershell in 2012 and 2016 doesn't have the right cmdlets. You could try upgrading PS on thos OS?

Why are you proposing the change in syntax? Were you getting problems with the old one? I ask because one niggle I ran into is the Key in the Zabbix template can't be too long or it won't accept it. Did this work for you when you tried it?

**Comment by matveevandrey** [comment]
Yes. Proposed command works and do not exceed key length constraint
